opengrok-web/src/main/java/org/opengrok/web/api/v1/controller/SystemController.java

```diff
@@ -74,4 +81,16 @@ public void loadPathDescriptions(@Valid final PathDescription[] descriptions) th
         ef.create(Set.of(descriptions), env.getDtagsEftarPath().toString());
         LOGGER.log(Level.INFO, "reloaded path descriptions with {0} entries", descriptions.length);
     }
+
+    @GET
+    @Path("/indextime")
+    @Produces(MediaType.APPLICATION_JSON)
+    public String getIndexTime() throws JsonProcessingException {
+        Date date = new IndexTimestamp().getDateForLastIndexRun();
+        ObjectMapper mapper = new ObjectMapper();
+        // StdDateFormat is ISO8601 since jackson 2.9
+        mapper.setDateFormat(new StdDateFormat().withColonInTimeZone(true));
+        String result = mapper.writeValueAsString(date);
```

Comment:
Why not return directly by `return mapper.writeValueAsString(date);` ? I think `result` variable name is too common to validate its use here.

Reply:
Separate variable could be handy for debugging however in this case it is probably unwarranted.